astropy/coordinates/erfa_astrom.py

```diff
@@ -105,6 +105,12 @@ def apio13(frame_or_coord):
             frame_or_coord.relative_humidity.value,
             frame_or_coord.obswl.value,
         )
+        # Note: this is inefficient. Really we should write our own apio13 version and
+        # strip out pvobs calls. However, we don't need a diurnal aberration/parallax
+        # term because we have already accounted for that in transforming to topocentric
+        # CIRS.
```

Comment:
I would suggest that you go ahead an do this as it only requires changing two lines of code in erfa.apio(). Otherwise, inertia may set in and might never get done.

Reply:
The main thing stopping me was not being able to work out how to make an empty astrom structure

Reply:
I still have to actually look at the PR, but making an empty structure is easy:
```
astrom = np.empty(shape, dtype=erfa.dt_eraASTROM)
```

Reply:
perfect - obvious now I see it. This might save some microseconds

Reply:
done